tests/system_test.py

```diff
@@ -421,29 +437,71 @@ def attributes(e, level):
             # top level list of tuples ('section-name', dict)
             return "".join(["%s {\n%s}\n"%(n, attributes(p, 1)) for n, p in self])
 
-    def __init__(self, name=None, config=Config(), pyinclude=None, wait=True,
+    def __init__(self, name=None, config=Config(), pyinclude=None, wait=True, wait_for_server_log=True,
```

Comment:
There are now three ways of waiting for server readiness. Wait for open port, wait for `/healtz` endpoint (if libwebsockets is enabled for compilation) or wait for special log line. It feels kind of confusing, what is the difference, when should I do what, and so on.